packages/manager/tools/webpack-config/src/webpack.common.ts

```diff
@@ -6,6 +6,9 @@ import CopyWebpackPlugin from 'copy-webpack-plugin';
 import get from 'lodash/get';
 import set from 'lodash/set';
 import MiniCssExtractPlugin from 'mini-css-extract-plugin';
+import TerserJSPlugin from 'terser-webpack-plugin';
```

Comment:
Why not `TerserWebpackPlugin` ?

Reply:
Roughly inspired by the webpack documentation (see: https://webpack.js.org/plugins/mini-css-extract-plugin/#minimizing-for-production).

Reply:
@antleblanc I don't understand, link should be https://webpack.js.org/plugins/terser-webpack-plugin/ right ?

Reply:
Please reach the section called "Minimizing For Production" (anchor doesn't work correctly).